app/controllers/actions/IdentifierAction.scala

```diff
@@ -70,6 +70,8 @@ class AuthenticatedIdentifierActionWithRegime @Inject() (
       .retrieve(Retrievals.internalId and Retrievals.allEnrolments and affinityGroup and credentialRole) {
         case _ ~ _ ~ Some(Agent) ~ _ =>
           Future.successful(Redirect(controllers.routes.UnauthorisedAgentController.onPageLoad(regime)))
+        case _ ~ enrolments ~ _ ~ Some(Assistant) if enrolments.enrolments.exists(_.key == "HMRC-MDR-ORG") =>
```

Comment:
@matthewharris1 Let the config choose the regime from application conf
keys {
  enrolmentKey.mdr = "HMRC-MDR-ORG"
}
```suggestion
        case _ ~ enrolments ~ _ ~ Some(Assistant) if enrolments.enrolments.exists(_.key == config.enrolmentKey(regime.toString)) =>
```

Reply:
Good idea fixed!